src/config.rs

```diff
@@ -626,6 +633,15 @@ impl Config {
             .unwrap_or(false)
     }
 
+    /// Whether to force plug update in Vim
+    pub fn force_vim_plug_update(&self) -> bool {
+        self.config_file
+            .vim
+            .as_ref()
+            .and_then(|c| c.force_plug_update)
+            .unwrap_or(false)
```

Comment:
```suggestion
            .unwrap_or_default()
```
is possible, but whatever you prefer